deno/rest/v10/guild.ts

```diff
@@ -158,6 +159,23 @@ export type RESTPostAPIGuildsJSONBody = AddUndefinedToPossiblyUndefinedPropertie
  */
 export type RESTPostAPIGuildsResult = APIGuild;
 
+/**
+ * https://discord.com/developers/docs/resources/guild#modify-guild-mfa-level
+ */
+export interface RESTPostAPIGuildsMFAJSONBody {
+	/**
+	 * MFA level
+	 *
+	 * See https://discord.com/developers/docs/resources/guild#guild-object-mfa-level
+	 */
+	level: GuildMFALevel;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/guild#modify-guild-mfa-level
+ */
+export type RESTPostAPIGuildsMFAResult = GuildMFALevel;
```

Comment:
Can someone check that the return type for this is not an object with a level property?